licenses/Apache-2.0.lre

```diff
@@ -3,6 +3,11 @@ Apache License 2.0
 https://spdx.org/licenses/Apache-2.0.json
 http://www.apache.org/licenses/LICENSE-2.0
 https://opensource.org/licenses/Apache-2.0
+
+exceptions:
+  (( i || properties ))		https://github.com/apache/rocketmq-client-go/issues/590
```

Comment:
Now that they've fixed this, can we drop this change?

Reply:
Not until they tag a new release. Their latest is still v2.0.0, which has "properties".

If you're uncomfortable adding it here I can keep it as an exception.